packages/developer-portal/src/sagas/revision-detail.ts

```diff
@@ -24,8 +24,8 @@ export const revisionDetailDataFetch = function*({
   try {
     const [response, scopes, desktopIntegrationTypes] = yield all([
       call(fetchAppRevisionsById, { id: appId, revisionId: appRevisionId }),
-      call(fetchScopesList),
-      call(fetchDesktopIntegrationTypesList, {}),
+      call(fetchScopeListAPI),
```

Comment:
Suggest to move this one to route-dispacher and fetch all from init app instead of fetch in sagas